templates/hiringPartnerPage.phtml

```diff
@@ -32,8 +32,34 @@
             <input id='companyUrl' name='companyUrl' class="submitHiringPartner form-control" type="text" placeholder="Company Website Url" data-max='255'>
             <input id="submitHiringPartner" type="submit" class="btn btn-success"/>
         </form>
-        <div id='messages'></div>
-        <h3>Current Hiring Partners</h3>
-        <div id='companies' ></div>
+    <div class="container">
+        <h1>Add contact to Hiring partner</h1>
+        <form id='addContactForm'>
+            <label for='companySize'>Company Size</label>
+            <select name='companySize' id='companySize' class='submitContact form-control'>
+                <option value='0'>Please select a company</option>
+                <?php echo \Portal\ViewHelpers\SelectCompanyViewHelper::selectCompanyDropdown($data); ?>
+<!--                    Above is a placeholder ViewHelper for the drop down-->
+            </select>
+            <label for="ContactName">name</label>
+            <input id="contactName" name="contactName" class="submitContact form-control" type="text" placeholder="Contact Name" data-required='true' data-max='600'>
+            <label for='email'>email</label>
+            <input id='email' name='email' class="submitContact form-control" type="text" placeholder="email" data-required='true' data-max='40'>
+            <label for='jobTitle'>Job title</label>
+            <input id='jobTitle' name='jobTitle' class="submitContact form-control" type="text" placeholder="Job title" data-max='255'>
+            <label for='contactPhoneNumber'>Phone Number</label>
+            <input id='contactPhoneNumber' name='contactPhoneNumber' class="submitContact form-control" type="text" placeholder="Phone Number" data-max='20'>
+            <div class="form-check">
+                <input class="form-check-input" type="checkbox" value="" id="isPrimaryContact">
+                <label class="form-check-label" for="defaultCheck1">
+                    Primary contact
+                </label>
+            </div>
+            <input id="submitHiringPartnerContact" type="submit" class="btn btn-success"/>
+        </form>
     </div>
+    <div id='messages'></div>
+    <h3>Current Hiring Partners</h3>
+    <div id='companies' ></div>
```

Comment:
Extra space after 'companies'

Reply:
There is still a space between 'companies' and the closing bracket >